### PR TITLE
loaded all data into dataframe'

### PR DESCRIPTION
**Pull request by VanditPatel1**
wtf @jerrguo i never got a review

### CODE REVIEW

File: get_curr.py [hunk 1]
```diff
@@ -12,8 +13,8 @@ def get_all_exhange_rates(all_combos):
 
 	all_combos = ','.join(all_combos) #join all currencies in list
 	rates = requests.get(BASE_URL+QUOTES+all_combos+API_TOKEN1) #get rates from API 
-	exhange_rates = json.loads(rates.content) #load into json format
-	all_info = json.dumps(exhange_rates, indent=4) #get visuals for print
+	all_info = json.loads(rates.content) #load into json format
```
Comment (jerrguo) [inline get_curr.py:16]: learn how to write code properly

**Review by jerrguo — APPROVED** [review]

File: get_curr.py (hunk 1 above)
Comment (Shehan29) [reply get_curr.py:16]: agreed

Comment (VanditPatel1) [reply get_curr.py:16]: idc im merging

Comment (VanditPatel1) [reply get_curr.py:16]: jk too many conflicts

**Pull request closed without merging**